shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
Well this works for me with the `Fedora.23` I provided the patch for (I already used the `s390x` so no need for change there):

```diff
diff --git a/shared/cfg/guest-hw.cfg b/shared/cfg/guest-hw.cfg
index fe55a79..0ccd337 100644
--- a/shared/cfg/guest-hw.cfg
+++ b/shared/cfg/guest-hw.cfg
@@ -43,7 +43,7 @@ variants:
             nic_model = virtio-net-device
             # Currently arm does not support msix vectors
             enable_msix_vectors = no
-        s390:
+        s390-virtio:
             # Currently s390 only supports virtio-net-ccw
             nic_model = virtio-net-ccw
             # Currently s390 does not support msix vectors
@@ -94,7 +94,7 @@ variants:
             drive_format=virtio-blk-device
             cd_format = scsi-cd
             scsi_hba = virtio-scsi-device
-        s390:
+        s390-virtio:
             # Currently s390 only supports virtio-blk-ccw on s390
             drive_format=virtio-blk-ccw
             cd_format = scsi-cd
diff --git a/shared/cfg/machines.cfg b/shared/cfg/machines.cfg
index 2959195..2555797 100644
--- a/shared/cfg/machines.cfg
+++ b/shared/cfg/machines.cfg
@@ -46,8 +46,8 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
-    - s390:
-        only s390
+    - s390-virtio:
+        only s390x
         auto_cpu_model = "no"
         cpu_model = host
         machine_type = s390-ccw-virtio
```

Where the `s390-virtio` is the avocado definition of the HW platform (in the future `s390-pci` or `s390-foobar` can be added) and the `s390x` is the guest os definition. Anyway I'd be fine with either  `s390` or `s390x`, the `s390x` looks more plausible to me as it's used in cd images.